validator_derive/src/validation.rs

```diff
@@ -123,13 +123,13 @@ pub fn extract_range_validation(
                     match ident.to_string().as_ref() {
                         "message" | "code" => continue,
                         "min" => {
-                            min = match lit_to_float(lit) {
+                            min = match lit_to_float_or_path(lit) {
                                 Some(s) => Some(s),
                                 None => error(lit.span(), "invalid argument type for `min` of `range` validator: only integers are allowed")
```

Comment:
we probably need to update the error messages if we allow float and not just integers...

Reply:
You're right, I missed that :).